packages/builders/src/index.ts

```diff
@@ -36,8 +36,6 @@ export * from './interactions/slashCommands/mixins/SharedSlashCommandOptions.js'
 export * as ContextMenuCommandAssertions from './interactions/contextMenuCommands/Assertions.js';
 export * from './interactions/contextMenuCommands/ContextMenuCommandBuilder.js';
 
-export * from './util/jsonEncodable.js';
```

Comment:
u r removing these 2 fns. So isn't it breaking? instead re-export `/util` package from builder and remove `_exportStar` from djs

Reply:
I added the star export for builders however, just to be explicit I'll keep the start export for `/util` in djs.